### PR TITLE
Fixes issue #15 : Added Dynamic Contributors Section

### LINKED ISSUE #15: Contributors Name in Readme
- Add dynamic contributor name in Readme File

### PR DESCRIPTION
**Pull request by AlacritousCreature**
# Description

This PR fixes issue #15 
Made changes in README.md file

Closes #15

## Type of change

- [X] This change requires a documentation update

# Checklist:

- [X ] My code follows the style guidelines of this project
- [X ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules

## Screenshots

 Updated screenshots
  
Screenshot (456)

---
Screenshot (457)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,3 +44,15 @@
 This project is completely Open source. Any willing to contribute can take up an issue or raise a new and work on it after getting assigned.
 
 ---
+## Contributors :v:
```
Comment (codewithvk) [inline README.md:47]: Please add it as the last section of readme file.

**Review by codewithvk — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (AlacritousCreature) [reply README.md:47]: Hey it's added in the last section of README.md, in these two screenshots above one is of the below section where that section is....and second image is of contents at the top of readme file......you may see it on my commited branch

Comment (codewithvk) [reply README.md:47]: apologies for that

**Comment by codewithvk** [comment]
Thanks for your PR @AlacritousCreature

**Comment by AlacritousCreature** [comment]
Thanks for merging @codewithvk !

**Pull request merged**